andela/ah-centauri-frontend
# #164829157 Setup React Router Version 4 or Higher

#### What does this PR do?
It setups react router version 4 or higher

#### Description of Tasks to be completed?
Installs and configures the router and creates the  Header, Footer, Login and Home components and links to the Semantic-UI-CSS in order to create a dummy route for login and the "/" page to test the configuration.

#### How should this be manually tested?
- Clone the project: `git clone https://github.com/andela/ah-centauri-frontend`
- cd into the project folder
- checkout to branch `chore/164829157-react-router`
- Install dependencies: `npm install`
- Launch development server: `npm start`
- Opens the homepage on the browser you should see the navbar, a Home text and the footer.
- navigate to the login page on the top right navigation bar

#### Type of change
 - [x] chore(a non-breaking change that sets up react router).

#### What are the relevant Pivotal Tracker Stories?
[#164829157](https://www.pivotaltracker.com/story/show/164829157)
